.travis.yml

```diff
@@ -0,0 +1,69 @@
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+dist: bionic
+language: go
+
+sudo: required
+services:
+  docker
+
+addons:
+  apt:
+    packages:
+      - protobuf-compiler
+      - python-protobuf
+
+branches:
+  only:
+    - travis
+
+go_import_path: github.com/hyperledger-labs/fabric-private-chaincode
+
+env:
+  global:
+    # SGX config
+    - SGX_MODE=SIM
+    - SGX_SDK=/opt/intel/sgxsdk
+    # SGX SSL
+    - OPENSSL=1.1.0j
```

Comment:
We might upgrade to 1.1.0l which is the latest version?  (btw: apropos version, bruno noticed that you forgot to update README.md from 1.4.2 to 1.4.3, so if you change versions, you might fix also README.md. While version changes could deserve a separate pr, you already changed the sdk version, so we might also update openssl version and fix other version-related omissions ...)

Reply:
Regarding fabric version, yes it seems that I didn't catch all 1.4.2 when updating README.md. Can one use variables in markdown?

Reply:
Upgraded openssl and added this to commit msg

Reply:
with openssl 1.1.0j CI-build goes through